api/lib/infrastructure/serializers/jsonapi/sco-organization-learner-serializer.js

```diff
@@ -16,6 +18,12 @@ const serializeWithUsernameGeneration = function (scoOrganizationLearner) {
 
 const serializeExternal = function (scoOrganizationLearner) {
   return new Serializer('external-users', {
+    transform(externalUser) {
+      return {
+        ...externalUser,
+        id: externalUser.id ?? randomUUID(),
```

Comment:
Compte tenu que l'externalUser est un JWT non-décodé, il n'y aura jamais de propriété `id` disponible dessus, alors autant ne pas suggérer qu'une propriété `id` pourrait exister :

```suggestion
        id: randomUUID(),
```

Reply:
Je pense que `externalUser` est la version décodé du JWT :)

Reply:
Tu me fais douter :thinking:

Reply:
J'ai le souvenir d'avoir vu le JWT dans un debug lors de mes tests.

Reply:
C'est fait